src/containers/Application/ApplicationList.tsx

```diff
@@ -23,29 +50,49 @@ const ApplicationList: React.FC = () => {
           <List.Item key={`app_selected_parameter_${value}`} content={key + ' : ' + value} />
         ))}
       </List>
-      {applications && applications.length > 0 && (
+      {userRole && headers && applications && applications.length > 0 && (
         // TODO: Implement sortable columns -> Requires reducer
```

Comment:
> Sorting should be done in the GraphQL right? 

Yes. But the sorted columns would be defined be what headers were clicked in the UI - which should be store in states (not reducers sorry). I'll update this comment.

Reply:
> Clicking a "sort" column should just update the query, just like changing a filter, or pagination settings. (so nothing fancy needs to be done to sort in here I would think)

Yes. That's right. We would update the URL and that would be used in the query. So, not even a state (and my previous comment).

Reply:
> Because the sorting should take into account the entire result set, not just the ones currently returned for the current page, right?

Yes, the comment was old. I'll update now trying to summary was was commented here.